src/beman/optional26/tests/test_types.hpp

```diff
@@ -50,7 +50,7 @@ struct derived : public base {
 
 struct move_detector {
     move_detector() = default;
-    move_detector(move_detector&& rhs) { rhs.been_moved = true; }
+    constexpr move_detector(move_detector&& rhs) { rhs.been_moved = true; }
```

Comment:
Should we add constexpr to other/all constructors in this class/file?

Reply:
The main reason for the duplication was that I duplicated all the tests to start with, and then worked to consteval them. It made it a little easier to track progress, but I wouldn't make it a mandatory pattern, necessarily. 
If it were consteval first, it might depend on implementation of the non-const version needs separate testing?